src/Postmark/PostmarkAdminClient.php

```diff
@@ -345,15 +345,25 @@ function deleteDomain($id) {
 	}
 
   /**
-	 * Request that the Postmark API verify the SPF records associated
-	 * with the Domain. Configuring SPF is not required to use
-	 * Postmark, but it is highly recommended, and can improve delivery rates.
+	 * Request that the Postmark API verify DKIM keys associated
+     * with the Domain.
+	 *
+	 * @param  integer $id The ID of the Domain we wish to verify DKIM keys on.
+	 * @return DynamicResponseModel
+	 */
+	function verifyDKIM($id) {
+		return new DynamicResponseModel($this->processRestRequest('PUT', "/domains/$id/verifyDkim"));
+	}
+
+  /**
+	 * Request that the Postmark API verify Return-Path DNS records associated
+     * with the Domain.
 	 *
-	 * @param  integer $id The ID for the Domain for which we wish to verify the SPF records.
+	 * @param  integer $id The ID of the Domain we wish to verify Return-Path DNS record on.
 	 * @return DynamicResponseModel
 	 */
-	function verifyDomainSPF($id) {
```

Comment:
Just to confirm: Are we removing a public feature here as well? 
Do we need to have a deprecation period or other "retirement strategy" in place?
I noticed there's also a mention of this particular method having taken place of _another_ deprecated method:

https://github.com/ActiveCampaign/postmark-php/blob/4ad4180c8ed60740ef1d888cb3ba2b1638080b7c/src/Postmark/PostmarkAdminClient.php#L259

Reply:
https://postmarkapp.com/developer/api/domains-api#verify-spf - yes we are removing this. I asked in the backend developer meeting and we should encourage people to use the new methods.